Nodejs/Product/Profiling/Profiling/StandaloneTargetView.cs

```diff
@@ -152,7 +152,7 @@ public StandaloneTargetView(StandaloneTarget template)
                     WorkingDirectory.IndexOfAny(Path.GetInvalidPathChars()) == -1 &&
                     (Path.IsPathRooted(ScriptPath) || Path.IsPathRooted(WorkingDirectory)) &&
                     File.Exists(Path.Combine(WorkingDirectory, ScriptPath)) &&
-                    (string.IsNullOrEmpty(WorkingDirectory) || Directory.Exists(WorkingDirectory)) &&
+                    (WorkingDirectory == string.Empty || Directory.Exists(WorkingDirectory)) &&
```

Comment:
We use WorkingDirectory at the line 152, before check it for null. So I can suppose that check is not needed. Maybe NullReferenceException is possible  at the line 152. Maybe we should check it for null before using.